draft-ietf-mmusic-sctp-sdp.xml

```diff
@@ -122,14 +122,15 @@
       values with the SDP Offer/Answer mechanism <xref target="RFC3264"/> for negotiating
       SCTP-over-DTLS associations.
   	</t>
+    <t>
+      NOTE: Due to the characteristics of TCP, usage of 'TCP/DTLS/SCTP' will always
+      force ordered and reliable delivery of the SCTP packets, which limits the usage of
+      the SCTP options. Therefore, it is strongly RECOMMENDED that TCP is only used
+      in situations where UDP traffic is blocked.
+    </t>
 		<t>
-			NOTE: TLS is designed to run on top of a byte-stream oriented
-			transport protocol providing a reliable, in-sequence delivery like
-			TCP. <xref target="RFC6083"/> presents serious limitations with transporting TLS
-			on top of SCTP. Therefore, defining a mechanism to negotiate media streams
-			transported using TLS on top of SCTP, i.e., 'SCTP/TLS', is outside the scope
-			of this specification. In addition, defining a mechanism to negotiate non-protected
-      SCTP associations is also outside the scope of this specification.
+			NOTE: The definition of a mechanism to negotiate non-protected SCTP associations
```

Comment:
Native SCTP is SCTP-over-IP. Non-protected SCTP is SCTP-over-IP as well as SCTP-over-UDP or SCTP-over-TCP. You can also have encryption on top of SCTP, so you can create SCTP/DTLS, TCP/SCTP/DTLS and UDP/SCTP/DTLS, all of which will be protected by encryption but not covered by this draft. Only thing this draft covers is SCTP encapsulated in DTLS. All other SCTP based protocol variants are note covered. 

I would prefer to remove this note or make it something like: This draft only defines the definition mechanism to negotiate SCTP-over-DTLS associations and does not cover other possible SCTP based protocols such as SCTP-over-IP, DTLS-over-SCTP-over-IP, SCTP-over-UDP, or SCTP-over TCP.